frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx

```diff
@@ -128,24 +128,28 @@ const DevPortfolioForm: React.FC = () => {
         contentMsg: 'Please select another dev portfolio.'
       });
     } else {
+      const finalPRs = openPRs.filter((pr) => pr !== '');
```

Comment:
What if we did this filtering higher up? Like above line 75-ish? That way we don't have to do any extra checking elsewhere.

Right now this logic misses the case where I have a blank "Opened PR" and "Reviewed PR" in my first field, and a link as my second field:
<img width="957" alt="Screenshot 2024-11-18 at 5 59 49 PM" src="https://github.com/user-attachments/assets/f1a994b6-eeff-4eb7-9b7e-f75eddf7a057">

Reply:
ohhh yeah I see, I'll fix that